modules/speech-to-text-impl/src/main/java/org/opencastproject/speechtotext/impl/engine/VoskEngine.java

```diff
@@ -120,8 +141,11 @@ public File generateSubtitlesFile(File mediaFile, File preparedOutputFile, Strin
     } finally {
       IoSupport.closeQuietly(process);
     }
+    Map<String,Object> returnValues = new HashMap<>();
+    returnValues.put("subFile",preparedOutputFile);
+    returnValues.put("language",language);
```

Comment:
Would probably be nice if we could define "subFile" and "language" as constants somewhere